### PR TITLE
Setup tox

### PR DESCRIPTION
**Pull request by YukieK**
This PR resolves issue #11.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,25 @@
 version: 2.1
 
 executors:
-    my-executor:
+    python35-executor:
```
Comment (mitsutaka) [inline .circleci/config.yml]: s/-executor//

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+        docker:
+            - image: circleci/python:3.5.6
+        environment:
+            - BASH_ENV: /tmp/bashrc
+    python36-executor:
```
Comment (mitsutaka) [inline .circleci/config.yml]: ditto

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+        docker:
+            - image: circleci/python:3.6.8
+        environment:
+            - BASH_ENV: /tmp/bashrc
+    python37-executor:
```
Comment (mitsutaka) [inline .circleci/config.yml]: ditto

File: .circleci/config.yml (hunk 1 above)
Comment (YukieK) [reply .circleci/config.yml]: Fixed to 'python35'.

File: .circleci/config.yml (hunk 2 above)
Comment (YukieK) [reply .circleci/config.yml]: Fixed to 'python36'.

File: .circleci/config.yml (hunk 3 above)
Comment (YukieK) [reply .circleci/config.yml]: Fixed to 'python37'.

**Pull request merged**